### ISSUE TITLE
Wrong Mail Font, Beta requester

### ISSUE THREAD

**Issue by username7291** [jens-maus/yam#713]
Can you use the old Texteditor.mcc(15.38) or tell it to use the right font? 
The new(15.46) uses topaz font and new yam refused to work with the old version.
With old yam(2010) and the new Texteditor its the same wrong font(topaz).
The Mails look ugly and stretched in width.

Can you remove the Beta requester at startup? The official Version is pretty unuseable cause web.de/gmx forces new TLS now, so all use the Beta and annoyed about the requester.

Exept of this good work no crash until now, i dont update yam for years cause some versions already crashed within 2 Minutes.

**Comment by javierdlr** [comment]
You mean TE v15.56?!?! Here it works fine, no topaz font as default, mayeb you have some topaz/strange setting under MUI TextEditor.
I'm using MUI5 nighlty build.

**Comment by username7291** [comment]
No, 15.46. Yam installer told me it install this new Version over the old one. No, MUI has no font settings. And if i replace the Texteditor with 15.38 i have again the good font with the old YAM Version. Im using MUI 19.35, maybe the new Texteditor create a incompatiblity with older MUI Versions. Or takes now a different Font from Workbench System settings, YAM takes Screen Font, and the NEW Texteditor use Default Font of System(this cant set to times). Texteditor 15.38 and below uses Screen Font like YAM do.

**Comment by Futaura** [comment]
I'm assuming this issue was fixed when using a more recent version of TextEditor.mcc? If not, please re-open.

Type the following into a shell to hide the requester:
`setenv save I_KNOW_YAM_IS_UNDER_DEVELOPMENT 1`
